src/types/params/dealer.ts

```diff
@@ -0,0 +1,19 @@
+import { PaginationParams } from "./index"
+import { DealerSortParams } from "../sort"
+
+export interface DealerLocationFilter {
+  cityId?: string
+  radius?: string
+}
+
+export interface DealerQueryParams {
+  feature: string[]
+  language?: string
```

Comment:
you might wanna specify what languages we have `"de" | "fr" | "it" | "en"`

Reply:
in all the other interfaces we don't specify. so i will leave it like it is